### ISSUE TITLE
Tree shaking not working on example repo

### ISSUE THREAD

**Issue by schrink** [Nothing-Works/vuetifyjs-mix-extension#24]
Hi, 

It seems to me that if I want to use 'vuetify-loader' it does not work for the components that are outside of .vue components. I have tried with your example repo here [https://github.com/Nothing-Works/mix](https://github.com/Nothing-Works/mix) and if I do something like this: 

```
// resources/views/welcome.blade.php
<div id="app">
        <v-app>
            <v-btn>
                Test
            </v-btn>
        </v-app>
        <example-component></example-component>
    </div>

```

It will give me an error: 
`[Vue warn]: Unknown custom element: <v-app> - did you register the component correctly? For recursive components, make sure to provide the "name" option.`

`[Vue warn]: Unknown custom element: <v-btn> - did you register the component correctly? For recursive components, make sure to provide the "name" option.`

Is this expected? 

Second, in this same repo, it does not seem to load all the styles properly. This is how it looks: 

![Laravel](https://user-images.githubusercontent.com/2532619/107855953-eb922180-6e25-11eb-8675-a0e3e0778a2b.png)

I installed everything as it is and no changes except the one I reference here. Am I doing something wrong, or this is a bug? 

Thanks!

**Comment by Nothing-Works** [comment]
@schrink I think the repo has not been updated for a long time, I will update it after fixing some bugs.

**Comment by schrink** [comment]
@Nothing-Works Yes, but the problems I explained are not only example repo related. I can’t make Vuetify work for components that are not within .vue files if I enable tree shaking. Otherwise it works normally. 

I used your repo just to isolate other possible reasons related to my project. This only confirms that this is most likely a bug. 

Btw, thank you for your awesome work!
